doc/folders.md

```diff
@@ -426,23 +426,24 @@ Get All Cascade Policies on Folder
 ----------------------------------
 
 To get a list of all cascade policies on a folder, which show the metadata templates that are being applied to all 
-items in the folder, call [`getAll(BoxAPIConnection api, String folderID)`][get-all] with the ID of the folder.
+items in the folder, call [`getCascadePolicies()`][get-all] on that folder.
 
 ```java
 String folderID = "2222";
-BoxMetadataCascadePolicy.getAll(api, folderID);
+BoxFolder folder = new BoxFolder(api, folderID);
+Iterator<BoxMetadataCascadePolicy.Info> metadataCascadePolicies = folder.getCascadePolicies().iterator();
```

Comment:
Most users aren't going to manually call `.iterator()` here — it might be more illustrative to show how to use this with a for-in loop